zstacklib/zstacklib/network/firewall/ip6tables.py

```
@@ -0,0 +1,473 @@
+                empty_chain_names.append(c.name)
+                c.delete()
+            return empty_chain_names
+
+        def _clean_rule_having_stale_target_chain() -> List[Node]:
+            """Clean rule having stale target chain."""
+            alive_chain_names = []
+            for t in self.children:
+                for c in t.children:
+                    alive_chain_names.append(c.name)
+
+            def walker(rule, data) -> bool:
```

Ruff (0.15.2)

[warning] Unused function argument: `data`

(ARG001)